components/Dropdown/dropdown.scss

```diff
@@ -1,111 +0,0 @@
-@import "colours/palette";
```

Comment:
any reason for leaving the empty file here?

Reply:
It's used in header.scss at the moment.

Reply:
sorry, didn't make myself clear.  You've cleared the contents of dropdown.css, but left the file in place.  I'm suggesting removing the file instead.

Reply:
Silly me.  Yeah, no reason at all.  Will remove, cheers.